src/doc/unstable-book/src/compiler-flags/wasm-c-abi.md

```diff
@@ -3,8 +3,8 @@
 This option controls whether Rust uses the spec-compliant C ABI when compiling
 for the `wasm32-unknown-unknown` target.
 
-This makes it possible to be ABI-compatible with all other spec-compliant Wasm
-like Rusts `wasm32-wasip1`.
+This makes it possible to be ABI-compatible with all other spec-compliant WASM targets
+like Rust's `wasm32-wasip1`.
```

Comment:
The use of any possessive here is incorrect. The naming scheme of `wasm32-wasip{N}` is also recognized by clang, it is not exclusive to us.
 #```suggestion
like `wasm32-wasip1`.
```

Reply:
Interesting, I see what you mean. In this context, does "ABI compatibility" mean the interoperability of two Wasm modules where one may be compiled for the `wasm32-unknown-unknown` target and the other `wasm32-wasip1`.

Reply:
@workingjubilee if you have a chance, may you PTAL at my question above?